remo/dashboard/views.py

```diff
@@ -272,3 +273,21 @@ def list_action_items(request):
                    'pageheader': pageheader,
                    'pageuser': user,
                    'query': request.GET.get('query', '')})
+
+
+def kpi(request):
+    countries = product_details.get_regions('en').values()
+    countries.sort()
+
+    categories = FunctionalArea.objects.all()
+
+    initiatives = Campaign.objects.all()
```

Comment:
Probably this should be replaced by 'active_objects' manager. The same also applies for the FunctionArea manager.